S2VX.Game/Editor/Reversible/ReversibleAddCommand.cs

```diff
@@ -1,19 +1,29 @@
-﻿using S2VX.Game.Story;
+﻿using S2VX.Game.Editor.Containers;
+using S2VX.Game.Story;
 using S2VX.Game.Story.Command;
 
 namespace S2VX.Game.Editor.Reversible {
     public class ReversibleAddCommand : IReversible {
-        public S2VXCommand Command { get; set; }
+        private S2VXCommand Command { get; set; }
 
         private S2VXStory Story { get; set; }
 
-        public ReversibleAddCommand(S2VXStory story, S2VXCommand command) {
+        private CommandPanel CommandPanel { get; set; }
+
+        public ReversibleAddCommand(S2VXStory story, S2VXCommand command, CommandPanel commandPanel) {
             Story = story;
             Command = command;
+            CommandPanel = commandPanel;
         }
 
-        public void Undo() => Story.RemoveCommand(Command);
+        public void Undo() {
+            Story.RemoveCommand(Command);
```

Comment:
For reusability, it seems like we can just call `CommandPanel.RemoveCommand(Command)` here. Not sure if there are side effects I'm not aware of though. `CommandPanel.LoadCommandLists()` can then be private. And also, we can reuse `CommandPanel.AddCommand(Command)` in other areas.

Reply:
Ah yes, I added those functions after I wrote this and didn't realize this could be changed to use them. Done.